package.json

```diff
@@ -12,7 +12,7 @@
     "build:lib": "babel --out-dir lib src",
     "build:flow": "flow-copy-source -v -i '{**/test/*.js,**/*.test.js}' src lib",
     "prebuild:dist": "rimraf dist/*",
-    "build:dist": "rollup -c && rollup -c --environment PRODUCTION",
+    "build:dist": "rollup -c && rollup -c --environment ESBUNDLE&& rollup -c --environment PRODUCTION",
```

Comment:
little nitpick: space is missing before &&
I don't know if some shells have a problem with it.